examples/basic-pnpm/pnpm-lock.yaml

```diff
@@ -1,13 +1,13 @@
-lockfileVersion: '6.1'
```

Comment:
@MikeMcC399 is there a reason to downgrade the lockfileVersion?

Reply:
Hi Ben @nagash77 and welcome back!

There was a bug in `pnpm` . See https://github.com/pnpm/pnpm/releases/tag/v8.6.2. I have followed the lead from `pnpm`

![image](https://github.com/cypress-io/github-action/assets/66998419/b5caef43-3d3e-4359-8359-66df3a95f3e5)

PS Sorry for my finger trouble. I got mixed up about which PR you had merged and I deleted this branch by mistake. It is restored and the PR is re-opened.